tools/PI/DevHome.PI/ViewModels/ExpandedViewControlViewModel.cs

```diff
@@ -234,4 +239,27 @@ public void Navigate()
         var navigationService = Application.Current.GetService<INavigationService>();
         navigationService.NavigateTo(Links[SelectedNavLinkIndex]?.PageViewModel?.FullName!);
     }
+
+    public void NavigateToSettings(string viewModelType)
+    {
+        // Because the Settings item isn't part of our NavLink list, when the user selects Settings,
+        // we need to move the list selection so that when they subsequently select an item from
+        // the NavLinks, we'll navigate to the correct page even if that was the previously-selected item.
```

Comment:
The screenshot in the PR shows Insights page selected when Settings is selected. Is it possible to not have a selected page?

Reply:
Yes, unfortunately this is an artifact of how our main nav list doesn't use a NavigationView. We have a backlog item to fix this all-up.

Reply:
sounds good